src/main/java/redis/clients/jedis/params/XPendingParams.java

```diff
@@ -30,16 +38,34 @@ public XPendingParams idle(long idle) {
     return this;
   }
 
+  /**
+   * @deprecated Use {@link #start(redis.clients.jedis.args.RangeEndpoint)}.
+   */
+  @Deprecated
   public XPendingParams start(StreamEntryID start) {
     this.start = start;
```

Comment:
```suggestion
    this.startEndpoint = RangeEndpoint.of(start);
```

Why do you need to deprecate it? it feels like in most cases it's easier to use it